src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -1793,7 +1888,11 @@ private void onDecodeOpen(
             // TODO: use buffer slot capacity instead
             this.encodeMaxFrameSize = Math.min(replySharedBudget, open.maxFrameSize());
             this.decodeMaxFrameSize = defaultMaxFrameSize;
+            this.encodeIdleTimeout = open.hasIdleTimeOut() ? open.idleTimeOut() : 0;
```

Comment:
If a `OPEN` frame doesn't have a `idleTimeout` (`!open.hasIdleTimeOut()`), do we still send the error? In that case, I need to update (add `idleTimeout` field) other spec scripts with `OPEN` frame.

Reply:
According to spec
>The idle timeout REQUIRED by the sender (see subsection 2.4.5). A value of zero is the same as if it was not set (null).

This would suggest that sending `0` amounts to not sending a required field, which would be a protocol error, agree?

Reply:
Actually, the spec is misleading - seems more like the `idle-timeout` behavior is _required_, but only when the `idle-timeout` field of `open` is specified and non-zero.